README.md

```diff
@@ -27,6 +27,10 @@ To get something usable on the EV3 brick, you should compile using [brickstrap].
     $ sudo apt-get install libgtk-3-dev
     # clone the git repo
     $ git clone git://github.com/ev3dev/ev3devKit
+    $ cd ev3devKit
+    $ git submodule init
```

Comment:
Most people do this in one command with `git submodule update --init`. In newer versions of git, you can actually just run `git clone --recursive  git://github.com/ev3dev/ev3devKit` to accomplish the same thing.